.github/assert-git-changes.py

```diff
@@ -4,13 +4,15 @@
 
 target_dir = os.environ.get("TARGET_DIR")
 
-if not target_dir:
-    print("No TARGET_DIR env variable provided. Exiting")
-    exit(1)
-
 git_changes = subprocess.getoutput("git status --porcelain")
-print(f"Changes:\n{git_changes}")
 
-if git_changes.find(target_dir) > 0:
-    print(f"Changes in {target_dir} detected! Failing")
-    exit(1)
+if git_changes:
+    print(f"Changes:\n{git_changes}")
+    if not target_dir:
+        print(f"Changes detected! Failing")
```

Comment:
I didn't get the idea of this change. :thinking: 

Previously the case of unset `target_dir` was handled with a more precise error message.

If we needed to handle storage layout changes by using this trick, then I would expect another more explicit approach, tbh.

Reply:
Yes and no))

Current implementation allow both specifying target subdir or analyzing from root. Here we need to analyze root.
As an option, we can explicitly use .storage-linter and rename parameter to `target`

Reply:
I am a bit wary of implicit assumptions here.

Reply:
`target_dir` was renamed to `target`